src/main/resources/assets/dynamiccrosshair/lang/ru_ru.json

```diff
@@ -0,0 +1,31 @@
+{
+  "text.autoconfig.dynamiccrosshair.title": "Dynamic Crosshair",
+
+  "text.autoconfig.dynamiccrosshair.option.dynamicCrosshair": "Динамически прятать прицел",
+  "text.autoconfig.dynamiccrosshair.option.dynamicCrosshair.@Tooltip": "Отображает прицел только если инструмент, оружие в руках или нацелен на сущность",
+  "text.autoconfig.dynamiccrosshair.option.dynamicCrosshairEnableWithHud": "Прицел при отображении интерфейса",
+  "text.autoconfig.dynamiccrosshair.option.dynamicCrosshairEnableWithHud.@Tooltip": "При выключении данной настройки прицел прячется при отключения интерфейса",
+  "text.autoconfig.dynamiccrosshair.option.dynamicCrosshairStyle": "Использовать стили прицела",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings": "Отображать прицел, когда:",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.onBlock": "Смотрит на блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.onEntity": "Смотрит на сущность",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.holdingTool": "Держит в руках инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.holdingRangedWeapon": "Держит в руках оружие дальнего боя",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.holdingThrowable": "Держит кидаемый предмет",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.holdingBlock": "Держит в руках блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairSettings.holdingUsableItem": "Держит в руках используемый предмет",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle": "Стили прицела",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.regular": "Стандартный",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.onBlock": "Смотрит на блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.onEntity": "Смотрит на сущность",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingTool": "Держит в руках инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingRangedWeapon": "Держит в руках оружие дальнего боя",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingThrowable": "Держит кидаемый предмет",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingBlock": "Держит в руках блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingUsableItem": "Держит в руках используемый предмет",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers": "Модификаторы прицела",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modInteractable": "Смотрит на интерактивный блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modCorrectTool": "Держит в руках подходящий инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modIncorrectTool": "Держит в руках неподходящий инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modUsableItem": "Holding usable item"
```

Comment:
Would you mind adding a translation to this one key that is still in English before I merge? :)